### ISSUE TITLE
Support for VS2019

### ISSUE THREAD

**Issue by Thieum** [zooba/indentguide#6]
Would be nice to have support for VS2019.

**Comment by Thieum** [comment]
Nevermind it's built-in since VS2017:
![image](https://user-images.githubusercontent.com/944894/52295069-de91ec00-2948-11e9-8763-cb8b72e465b6.png)

**Comment by ygoe** [comment]
Yes, it's built in but way less elegant and useful. The Microsoft lines are dashed not dotted, can't be highlighted (makes them almost useless), not properly aligned at the column edge and don't support additional right border lines. This extension is so much better. Too bad it seems to be dead and doesn't build anymore with today's tools.

**Comment by Thieum** [comment]
@ygoe for right border lines, check https://github.com/pharring/EditorGuidelines
